data_steward/cdr_cleaner/cleaning_rules/truncate_rdr_using_date.py

```diff
@@ -142,22 +157,52 @@ def get_sandbox_tablenames(self):
         return sandbox_tables
 
 
+def validate_date_string(date_string):
+    """
+    Validates the date string is a valid date in the YYYY-MM-DD format.
+
+    If the string is valid, it returns the string.  Otherwise, it raises either
+    a ValueError or TypeError.
+
+    :param date_string: The string to validate
+
+    :return:  a valid date string
+    :raises:  A ValueError if the date string is not a valid date or
+        doesn't conform to the specified format.
+    """
+    datetime.strptime(date_string, '%Y-%m-%d')
+    return date_string
```

Comment:
I think it makes sense to put this in a utility  somewhere but I'm not sure where.  Suggestions are appreciated.

Reply:
the ideal place to have this would be the utils module, which we do not have right now. For now, this can be stored in common I think.

Reply:
I looked in common and didn't see any function definitions in there, so that didn't feel right to me.  This is kind of a BigQuery thing.  Would it make sense to put this in `utils/bq.py`?